astropy/io/ascii/latex.py

```diff
@@ -310,16 +276,14 @@ def __init__(self, ignore_latex_commands=['hline', 'vspace', 'tableline'],
         self.data.comment = self.header.comment
 
     def write(self, table=None):
-        self.header.start_line = None
-        self.data.start_line = None
+        self.header.start_line = 0
+        self.data.start_line = 0
```

Comment:
@MSeifert04 this line remove the nonetype error i.e none was rreturn when start was not found and added to int